_docs/ops/security-ir.md

```diff
@@ -250,4 +254,4 @@ Guidelines for addressing Low-sev issues:
 * [Recent document history](https://github.com/cloud-gov/cg-site/commits/master/{{ page.path }}) (since 2020-02-05)
 * [Older document history](https://github.com/cloud-gov/cg-site/commits/master/content/docs/ops/{{ page.slug }}.md) (before 2020-02-05)
 
-[FedRAMP ISSO TRR reps]: (https://docs.google.com/document/d/1jGddQkjkQ6e9B0UTq9hfQqHe0btAbTeBGL_DxkozAcg/edit)
\ No newline at end of file
+[FedRAMP ISSO TRR reps]: (https://docs.google.com/document/d/1jGddQkjkQ6e9B0UTq9hfQqHe0btAbTeBGL_DxkozAcg/edit)
```

Comment:
```suggestion
[FedRAMP ISSO TRR reps]: https://docs.google.com/document/d/1jGddQkjkQ6e9B0UTq9hfQqHe0btAbTeBGL_DxkozAcg/edit
```

I actually have #2216 open to fix this issue, but I can close it if you make this change. Or if you approve it then I can merge it before yours